War Within update PTR

Thrumbar opened the pull request.

samyonair commented:
Have assigned Rythal to it, because he should make a decisson about it, sorry for merge failure by itallian language merge

nelegalno commented:
np, I believe the translations should take priority.
And it's a good idea to get Rythal's opinion on the .toc and Carbonite/NxTravel.lua file changes.

nelegalno commented:
Updated:
whitespace
Simplified translation files
Removed unused strings from Carbonite/Bindings.xml
Added missing line in [Carbonite.Quests/Locales/deDE.lua#L20] (https://github.com/nelegalno/Carbonite/blob/N2C14/Carbonite.Quests/Locales/deDE.lua#L20)

P.S. #271 shouldn't cause merge conflict.

samyonair commented:
hi nelegalno just for testing purpose and development have created branch DEV, can you please make a pull request there? 

Because from my understandig Rythal is out by now

Greetings Samyonair

Thrumbar committed:
Update phase 1 ( see description )

Updates
C_Item.
C_AddOns.
C_Reputation.
C_PvP.
nterfaceOptionsFrame_O to Settings.
C_Spell.
GetRidingSkill changes
GetMouseFocus to GetMouseFoci

Thrumbar committed:
Patch for following error....   :goodgrief:

1x bad argument #1 to '?' (Usage: local spellInfo = C_Spell.GetSpellInfo(spellIdentifier))
[string "=[C]"]: in function GetSpellInfo'
[string "@Carbonite/NxTravel.lua"]:917: in function GetRidingSkill'
[string "@Carbonite.Warehouse/NxWarehouse.lua"]:3339: in function func'
[string "@Carbonite/Libs/AceTimer-3.0-17/AceTimer-3.0.lua"]:57: in function <Carbonite/Libs/AceTimer-3.0/AceTimer-3.0.lua:50>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: local spellInfo = C_Spell.GetSpellInfo(spellIdentifier))"

Thrumbar committed:
attempt to call global 'ReputationFrame_Update' (a nil value)

And I cannot do anything small or simple, here it is...

Thrumbar committed:
Watch list item fix
